packages/account/src/Sections/Security/DeactivateAccount/deactivate-account-reason.jsx

```diff
@@ -126,26 +126,26 @@ class DeactivateAccountReason extends React.Component {
     validateFields = values => {
         const error = {};
         const selected_reason_count = selectedReasons(values).length;
+
         if (!selected_reason_count) {
             error.empty_reason = localize('Please select at least one reason');
         }
-        if ((values.otherTradingPlatforms + values.doToImprove).length > 0 || selected_reason_count) {
-            const max_characters = character_limit_no;
+        if ((values.other_trading_platforms + values.do_to_improve).length > 0 || selected_reason_count) {
             const final_value = preparingReason(values);
 
-            const remaining_characters = max_characters - final_value.length;
-            if (remaining_characters >= 0) {
-                this.setState({ remaining_characters });
-            } else {
-                this.setState({ remaining_characters: 0 });
-            }
+            const text_inputs_length = values.other_trading_platforms.length + values.do_to_improve.length;
+            let remaining_characters = character_limit_no - text_inputs_length;
+
+            remaining_characters = remaining_characters >= 0 ? remaining_characters : 0;
+
+            this.setState({ remaining_characters });
 
-            if (!/^[0-9A-z .,'-]*$/.test(final_value)) {
+            if (!/^[0-9A-z .,'-’]*$/.test(final_value)) {
                 error.characters_limits = localize("Must be numbers, letters, and special characters . , ' -");
             }
 
-            if (final_value.length > max_characters) {
-                error.characters_count_exceed = 'Please enter no more than 255 characters for both fields.';
+            if (remaining_characters.length > character_limit_no) {
+                error.characters_count_exceed = localize(`Please enter no more than  ${character_limit_no} characters for both fields.`);
```

Comment:
This condition won't go through, please help to change

Reply:
@prince-deriv, you may refer to here: https://github.com/binary-com/deriv-app/tree/master/packages/translations#usage for `Localize` usage, @matin-binary , we should add `localize` to that README as well.

For the change above, here's a suggestion:
`localize('Please enter no more than {{ char_limit }} characters for both fields.', { char_limit: character_limit_no } )`

Reply:
@prince-deriv , this needs to be resolve before we can move it forward.